app/src/main/java/ai/elimu/appstore/synchronization/AppSynchronizationActivity.java

```diff
@@ -163,6 +166,9 @@ public void run() {
                                 }
                             });
                             t.printStackTrace();
```

Comment:
Not directly related to this PR, but can you replace `t.printStackTrace();` with Timber logging?

Reply:
Yes sure.